src/language-js/embed.js

```diff
@@ -616,8 +611,27 @@ function printHtmlTemplateLiteral(
     }
   );
 
+  const firstWhitespace = node.quasis[0].value.raw.match(/^\s/) ? " " : "";
+  const lastWhitespace = node.quasis[node.quasis.length - 1].value.raw.match(
+    /\s$/
+  )
+    ? " "
+    : "";
+
+  if (firstWhitespace && lastWhitespace) {
```

Comment:
I think we wanted to do `|| options.htmlWhitespaceSensitivity === 'ignore'` here.